src/Utils.cpp

```diff
@@ -805,13 +805,28 @@ bool CTable::SetCell(const CString& sColumn, const CString& sValue,
     return true;
 }
 
-bool CTable::GetLine(unsigned int uIdx, CString& sLine) const {
+bool CTable::GetLine(unsigned int uIdx, CString& sLine, bool as_deflist) const {
     std::stringstream ssRet;
 
     if (empty()) {
         return false;
     }
 
+    if (as_deflist) {
+        if (m_vsHeaders.size() > 2) return false; // definition list mode can only do up to two columns
+        if (uIdx >= size()) return false;
+
+        const std::vector<CString>& mRow = (*this)[uIdx];
```

Comment:
nit: `VString` is an alias for that.

What does `m` stand for?

Reply:
all those variables are pre-existing ones. Those lines are adapted from the originals a few more lines down